boards/arm/nrf9160dk_nrf9160/doc/index.rst

```diff
@@ -156,6 +156,10 @@ image. The Secure image can be built using either Zephyr or
 `Trusted Firmware M`_ (TF-M). Non-Secure firmware images are always built
 using Zephyr. The two alternatives are described below.
 
+.. note::
+
+   By default the the Secure image for nRF9160 is built using TF-M.
```

Comment:
the ~the~

Reply:
done. (also in one more place, there was a double "the")